src/stories/Button.js

```diff
@@ -1,5 +1,5 @@
 import { html } from 'lit-html';
-import './button.css';
+// import './button.css';
```

Comment:
Not sure how you intend to build these tools out, but instead of trying to think of a way to "build" you code, I just skipped all the code that would have needed to be built, instead.